src/controller_examples/simple_controller/proof/safety.rs

```diff
@@ -140,6 +128,39 @@ proof fn next_preserves_reconcile_get_cr_done_implies_pending_req_in_flight_or_r
     }
 }
 
+pub proof fn next_and_not_crash_preserves_init_pc_or_reconciler_at_after_get_cr_pc_and_pending_req_in_flight_and_no_resp_in_flight(cr: CustomResourceView, s: State<SimpleReconcileState>, s_prime: State<SimpleReconcileState>)
```

Comment:
I think we should just move it to liveness.rs. Also, are all these assertions necessary?

Reply:
> Also, are all these assertions necessary?

It seems so.

But TBH I don't wanna use `assert(false)`, but how to avoid this?

Reply:
I remove several assertions here.

Reply:
You can try to prove forall !xxx